### PR TITLE
Upload GTK runtime zip after build

### LINKED ISSUE #145: Provide prebuilt releases
Having prebuilt releases for each gtk version would be cool. Those could be built via appveyor. What do you think?

### PR DESCRIPTION
**Pull request by danyeaw**
Closes #145 by zipping up the built runtime and uploading it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,36 @@
 This python script helps you build a full [GTK](https://www.gtk.org/) library
 stack for Windows using Visual Studio. Currently, GTK 3 and GTK 4 are supported.
 
+## Install GTK Only
+
+If you want to only run GTK on Windows and not build it yourself, you can download
+a zip file from the last release and unzip it to `C:\gtk`.
+
+It comes with GTK, Cairo, PyGObject, Pycairo, GtkSourceView, adwaita-icon-theme and
```
Comment (pbor) [inline README.md:13]: This should probably specify which gtk (3 or 4).

As discussed before the selection here is a bit arbitrary, but if it solves someone's problem I am ok with it

Comment (danyeaw) [reply README.md]: I updated the README to add more clarity. I think this is a start, and if others would like us to include other libraries we can in the future.

**Comment by danyeaw** [comment]
Hi @nacho and @pbor, any thoughts on this?

**Comment by danyeaw** [comment]
@pbor I added a statement that the binaries are provided as-is. What do you think?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,39 @@
...
 
+## Install GTK Only
+
+If you want to only run GTK on Windows and not build it yourself, you can download
+a zip file from the last release and unzip it to `C:\gtk`.
+
+It comes with GTK4, Cairo, PyGObject, Pycairo, GtkSourceView5, adwaita-icon-theme, and
+all of their dependencies.
+
+Unfortunately, these binaries are provided as-is, and we can't guarantee timely updates
```
Comment (pbor) [inline README.md:16]: I think we need a stronger statement, eg:

Note however that these binaries are provided “AS IS”, WITHOUT WARRANTY OF ANY KIND.
They just contain the output of our latest CI run. They are not tested and we cannot commit to timely updates even for security issues. We strongly recommend to build your own binaries, especially if you plan to distribute them with your application or use them in production.

**Review by pbor — COMMENTED** [review]
I am still a bit cagey with the whole idea of distributing binaries, but if you believe it is useful let's get this in.

File: README.md (hunk 2 above)
Comment (danyeaw) [reply README.md]: Hi @pbor, I updated the statement to match your wording.

**Pull request merged**